src/helpers.js

```diff
@@ -5,52 +5,42 @@ const TEXT_NODE = 3
 
 // Currently this fn only supports jest timers, but it could support other test runners in the future.
 function runWithRealTimers(callback) {
-  const fakeTimersType = getJestFakeTimersType()
-  if (fakeTimersType) {
+  return _runWithRealTimers(callback).callbackReturnValue
+}
+
+function _runWithRealTimers(callback) {
+  const timerAPI = {
+    clearImmediate,
+    clearInterval,
+    clearTimeout,
+    setImmediate,
+    setInterval,
+    setTimeout,
+  }
+
+  // istanbul ignore else
+  if (jest) {
     jest.useRealTimers()
   }
```

Comment:
Could we use this or does istanbul not support it?
```suggestion
  jest?.useRealTimers()
```

Reply:
Ah yes, it doesn't seem to check that syntax

Reply:
Do we know that this function isn't entered in an environment where `jest` is not defined i.e. are we sure this doesn't throw a ReferenceError?

Reply:
That's why I used `jest?.` If Jest is undefined, nothing will happen.

Reply:
You're right, both these examples would throw a `ReferenceError`. Instead we should check that:
```js
  if (typeof jest !== undefined) {
    jest.useRealTimers()
  }
```

Reply:
Good catch.
I've pushed this as a replacement:
```js
  try {
    jest.useRealTimers()
  } catch (e) {
    // not running in jest environment
  }
```

Reply:
I would still prefer the `typeof` check so we aren't swallowing errors from `useRealTimers`.